client/blocks/login/two-factor-authentication/two-factor-actions.scss

```diff
@@ -1,9 +1,27 @@
 .two-factor-authentication__actions.card {
 	margin-bottom: 0;
+	max-width: 400px;
```

Comment:
I think it might make sense to have this declared at the same place where we declare it for the login main container:

https://github.com/Automattic/wp-calypso/blob/c29ba2e5d238cc0c2207e56a7636bc987b9cf4b2/client/login/wp-login/style.scss#L33

That way they'll always be equal and changed together.